src/main/java/com/github/eyefloaters/console/api/model/ConfigEntry.java

```diff
@@ -63,6 +63,25 @@ public static ConfigEntry fromKafkaModel(org.apache.kafka.clients.admin.ConfigEn
         return entry;
     }
 
+    /**
+     * Compare the {@code value} field of this ConfigEntry with the other ConfigEntry's
+     * {@code value} field.
+     *
+     * <p>Numeric fields will be parsed before comparison, passwords will always compare
+     * as equal, and other types will be compared as strings.
```

Comment:
Maybe explain why passwords will always compare as equal.

Reply:
I updated the comment to indicate passwords are considered equal to avoid any information leakage. I think in practice the Admin client will always return `null` for configs of type password anyway.